### PR TITLE
MediaTypeNames+Application - update documentation

### PR DESCRIPTION
**Pull request by aik-jahoda**
## Summary

Collect all changes from #3916 related to MediaTypeNames+Application

### CODE REVIEW

File: xml/System.Net.Mime/MediaTypeNames+Application.xml [hunk 1]
```diff
@@ -64,8 +64,15 @@
...
-        <remarks>To be added.</remarks>
+        <summary>Specifies that the <see cref="T:System.Net.Mime.MediaTypeNames.Application" /> data is in JSON format.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The <xref:System.Net.Mime.MediaTypeNames.Application.Json> member designates that the data is in JSON format.
+  
+ ]]></format>
+        </remarks>
```
Comment (karelz) [inline xml/System.Net.Mime/MediaTypeNames+Application.xml:75]:
```suggestion
        <remarks>To be added.</remarks>
```

File: xml/System.Net.Mime/MediaTypeNames+Application.xml [hunk 2]
```diff
@@ -233,8 +240,15 @@
...
-        <remarks>To be added.</remarks>
+        <summary>Specifies that the <see cref="T:System.Net.Mime.MediaTypeNames.Application" /> data is in XML format.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The <xref:System.Net.Mime.MediaTypeNames.Application.Xml> member designates that the data is in XML format.
+  
+ ]]></format>
+        </remarks>
```
Comment (karelz) [inline xml/System.Net.Mime/MediaTypeNames+Application.xml:251]:
```suggestion
        <remarks>To be added.</remarks>
```

File: xml/System.Net.Mime/MediaTypeNames+Application.xml (hunk 1 above)
Comment (karelz) [reply xml/System.Net.Mime/MediaTypeNames+Application.xml:75]: Looks like duplicate text to summary - other formats like PDF, RTF, ZIP follow the same pattern

**Comment by aik-jahoda** [comment]
@karelz, please can you take a look? The PR is blocked by the requested changes.

**Pull request merged**